modules/extension/xsd/xsd-wcs/src/main/resources/org/geotools/wcs/v1_1/gml4wcs.xsd

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schema targetNamespace="http://www.opengis.net/gml" 
+xmlns:gml="http://www.opengis.net/gml"
+xmlns="http://www.w3.org/2001/XMLSchema" 
+elementFormDefault="qualified" version="1.1.1">
+	<annotation>
+		<appinfo source="urn:opengis:specification:gml:schema-xsd:gml:3.1.1">gml.xsd</appinfo>
+		<documentation>GML profile for WCS 1.2. Primary editor: Arliss Whiteside. Last updated 2007-06-06
+		Copyright © 2007 Open Geospatial Consortium, Inc. All Rights Reserved. </documentation>
+	</annotation>
```

Comment:
These are the actual schemas from the OGC, we track external (non LGPL) stuff like this in a review.MD file in the root of the module directory. Yeah I know the developer guide says /site/resources/review.txt but we moved to git hub.

Here is an example:

* https://github.com/geotools/geotools/blob/master/modules/extension/graph/REVIEW.md